CHANGES_1.1.txt

```diff
@@ -2449,7 +2449,7 @@ CHANGES IN FLTK 1.1.0b4                                 BETA: ??? ?? ????
 
   - Updated the flags_ member of Fl_Widget to be an
     integer instead of uchar, to support the new
-    FL_OVERRIDE flag for Fl_Window.
+    OVERRIDE flag for Fl_Window.
```

Comment:
Please revert this change.
This file (CHANGES_1.1.txt) should NOT be changed, it's for reference only.

Reply:
Oha, I just downloaded 1.1 and the flag was indeed FL_OVERRIDE. I did not remember that and thought I fixed old docs.